src/main/java/hu/oe/nik/szfmv/automatedcar/AutomatedCar.java

```diff
@@ -38,21 +38,8 @@ public PowerTrain getPowerTrain() {
     private void calculatePositionAndOrientation() {
         //TODO it is just a fake implementation
 
-        switch (virtualFunctionBus.samplePacket.getKey()) {
-            case 0:
-                y -= 5;
-                break;
-            case 1:
-                x += 5;
-                break;
-            case 2:
-                y += 5;
-                break;
-            case 3:
-                x -= 5;
-                break;
-            default:
-                break;
-        }
+        y -= virtualFunctionBus.toPowerTrainPacket.getGasPedalValue()/10;
```

Comment:
Kódformázási előírás nincs betartva.

Reply:
Beállítottam és lefuttattam a fájlokon a Save Actions (ütettem pár entert majd rámentettem, hogy lefusson).  Kijavítottam sok formázási hibát is,  a maradék túlnyomó része Magic Number az elemek elhelyezésénél a Dashboardon.